extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/sql/SqlMSQStatementResourcePostTest.java

```diff
@@ -273,12 +273,12 @@ public void durableStorageDisabledTest()
         NilStorageConnector.getInstance()
     );
 
-    String errorMessage = "The SQL Statement API cannot read from the select destination [DURABLE_STORAGE] provided in "
-                          + "the query context [selectDestination] since it is not configured. It is recommended to "
-                          + "configure the durable storage as it allows the user to fetch large result sets. "
+    String errorMessage = "The sql statement api cannot read from the select destination [durableStorage] provided in "
+                          + "the query context [selectDestination] since it is not configured on the [broker]. It is recommended to "
```

Comment:
Using `[broker]` doesn't seem right. We should either write it as:
```
the query context [selectDestination] since it is not configured on the node [broker].
```
or remove the interpolation
```
the query context [selectDestination] since it is not configured on the broker.
```
See point (2) here
https://github.com/apache/druid/blob/1ddbaa874429d6f176c90f25189c959d5d1eae55/dev/style-conventions.md?plain=1#L48